core/src/main/java/com/yahoo/ycsb/generator/FileGenerator.java

```diff
@@ -57,15 +58,14 @@ public synchronized String nextString()
 			System.err.println("NullPointerException: " + filename + ':' + current);
 			throw e;
 		} catch(IOException e) {
-			System.err.println("Exception: " + e);
-			return null;
-		}
+		  throw new RuntimeException(e);		}
```

Comment:
Are these errors caught somewhere else? Is there a reason that exceptions were just printed instead of being thrown previously?